recursively_regenerate_jlls.jl

```diff
@@ -38,7 +38,9 @@ function get_json_obj(dep_name::String)
     m = Module(:__anon__)
```

Comment:
Drive-by comment here, but this should probably be `Module(gensym())`?

Reply:
Shouldn't matter too much in this simple script, but in general probably yes, so here we go: #1684